### DISCUSSION TITLE
Error saving page after editing

### DISCUSSION THREAD

**Discussion by jontheronza** [givanz/Vvveb#131] [Q&A]
Hi, I have installed the latest version over 10 times now - the minute I go and edit a page, and I click on Save - I get an error saying "Error Saving!". The folder and files have permissions set to 777.

This happens with every template - even the default Landing template.

**Comment by givanz** [comment]
Hi

Can you please check `admin > system tools > error log` or php error log directly and see if there are any error messages? http://demo.vvveb.com/admin/index.php?module=tools/errorlog

Can you provide some info about the setup like os, php version?

Are you using latest.zip or github code?

**Reply by jontheronza** [reply]
Hi 

I have tried PHP versions 7.3, 7.4 and 8.0 - Server is Linux. For setup I am using latest.zip

This is from the server error log:

2024-06-12 10:04:01.768797 [NOTICE] [3087230] [T1] [196.210.77.210:51558#APVH_www.launchpoint.co.za] [MODSEC] mod_security rule [id "77316844"] at [/etc/modsecurity.d/001_i360_basic.conf:27] triggered! 
[Wed Jun 12 10:04:01.767027 2024] [error] [client 196.210.77.210] ModSecurity: Access denied with code 403, [Rule: 'ARGS' '@pmFromFile bl_db_domains'] [id "77316844"] [msg "IM360 WAF: Block by known DB injection entry||Injection:

**Comment by jontheronza** [comment]
Hi - I managed to get it working. Having looked at the error above with Modsecurity - I created a rule using the id "77316844" that disables that specific rule.

**Reply by givanz** [reply]
Hi

Thanks for feedback.

This is a common issue with mod_security, on save the content of the page is sent through POST and it often exceeds the size of `SecRequestBodyLimit` from apache mod_security.

A common workaround is to increase `SecRequestBodyLimit` to a larger value.
